nucypher/characters/chaotic.py

```diff
@@ -46,8 +47,7 @@ class MonitoringTracker(FleetStateTracker):
         def record_fleet_state(self, *args, **kwargs):
             new_state_or_none = super().record_fleet_state(*args, **kwargs)
             if new_state_or_none:
-                checksum, new_state = new_state_or_none
-                hey_joe.send({checksum: self.abridged_state_details(new_state)}, "states")
+                hey_joe.send(None, "states")
```

Comment:
Sending `None` over the websocket?  What's going on here?

Reply:
`None` is the message payload. I don't really need to pass any data to the subscriber (i.e. the UI), just a notification that the status UI page for Moe should update its 'states', based on a change. Same idea for 'nodes' notification. The UI will then know to update what is being shown.